src/dymaptic.GeoBlazor.Core/Components/Layers/FeatureLayer.cs

```diff
@@ -1055,28 +1065,60 @@ public class Attachment
     ///     The globalId of the attachment to be added or updated. These Global IDs must be from the Global ID field created by ArcGIS. For more information on ArcGIS generated Global IDs, see the Global IDs and Attachments and relationship classes sections in the <a target="_blank" href="https://enterprise.arcgis.com/en/server/latest/publish-services/windows/prepare-data-for-offline-use.htm#ESRI_SECTION1_CDC34577197B43A980E4B5021DB1C32A">Data Preparation</a> documentation.
     /// </summary>
     public string GlobalId { get; set; } = default!;
-    
+
     /// <summary>
     ///     The name of the attachment. This parameter must be set if the attachment type is Blob.
     /// </summary>
     public string? Name { get; set; }
-    
+
     /// <summary>
     ///     The content type of the attachment. For example, 'image/jpeg'. See the ArcGIS REST API documentation for more information on supported attachment types.
     /// </summary>
     public string? ContentType { get; set; }
-    
+
     /// <summary>
     ///     The id of pre-loaded attachment.
     /// </summary>
     public string? UploadId { get; set; }
-    
+
     /// <summary>
     ///     The attachment data.
     /// </summary>
     public string? Data { get; set; }
 }
 
+/// <summary>
+/// Time info represents the temporal data of a time-aware layer. The time info class provides information such
+/// as date fields that store the start and end times for each feature and the total time span for the layer.
+/// </summary>
+public class TimeInfo
```

Comment:
If these do always come from the feature service via JS, this could be a `record`, although with the xml comments, it wouldn't necessarily save any space.